checks/raw/security_policy.go

```diff
@@ -74,7 +74,9 @@ func SecurityPolicy(c *checker.CheckRequest) (checker.SecurityPolicyData, error)
 	case errors.Is(err, sce.ErrRepoUnreachable), errors.Is(err, clients.ErrUnsupportedFeature):
 		break
 	default:
-		return checker.SecurityPolicyData{}, err
+		if !errors.Is(err, clients.ErrUnsupportedFeature) {
+			return checker.SecurityPolicyData{}, err
+		}
```

Comment:
does the case above `default` not already handle `clients.ErrUnsupportedFeature`?

Reply:
I honestly have no clue why i put that there